packages/devextreme/scss/widgets/fluent/gridBase/_colors.scss

```diff
@@ -185,6 +185,7 @@ $datagrid-icon-link-disabled-opacity: $button-disabled-icon-opacity;
   $datagrid-row-alternation-bg: darken($datagrid-base-background-color, 4%) !default;
   $datagrid-text-stub-background-image-path: data-uri('images/widgets/fluent/color-schemes/light/grid/text-stub.png') !default;
   $datagrid-cell-removed-text-color: $datagrid-columnchooser-item-color !default;
+  $datagrid-nodata-color: #707070 !default;
```

Comment:
It's better to use '$base-label-color' variable instead of hardcore values of colors